openupgrade_scripts/scripts/stock/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,214 @@
+stock        / stock.route              / supplied_wh_id (many2one)     : NEW relation: stock.warehouse
+stock        / stock.route              / supplier_wh_id (many2one)     : NEW relation: stock.warehouse
+stock        / stock.route              / warehouse_ids (many2many)     : NEW relation: stock.warehouse
+stock        / stock.route              / warehouse_selectable (boolean): NEW
+# NOTHING TO DO: handle by ORM
+
+stock        / stock.rule               / location_dest_id (many2one)   : NEW relation: stock.location, required
+stock        / stock.rule               / location_id (many2one)        : DEL relation: stock.location, required
+# DONE: pre-migration: renamed columns
+
+stock        / stock.rule               / route_id (many2one)           : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+stock        / stock.scrap              / lot_id (many2one)             : relation is now 'stock.lot' ('stock.production.lot') [nothing to do]
+# NOTHING TO DO
+
+stock        / stock.scrap              / scrap_qty (float)             : now a function
+# NOTHING TO DO: handle by ORM
+
+stock        / stock.warehouse          / crossdock_route_id (many2one) : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+stock        / stock.warehouse          / delivery_route_id (many2one)  : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+stock        / stock.warehouse          / reception_route_id (many2one) : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+stock        / stock.warehouse          / resupply_route_ids (one2many) : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+stock        / stock.warehouse          / route_ids (many2many)         : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+stock        / stock.warehouse.orderpoint / route_id (many2one)           : relation is now 'stock.route' ('stock.location.route') [nothing to do]
+# NOTHING TO DO
+
+---XML records in module 'stock'---
+NEW ir.actions.act_window: stock.action_product_stock_view
+DEL ir.actions.act_window: stock.report_stock_quantity_action
+DEL ir.actions.act_window: stock.report_stock_quantity_action_product
+NEW ir.actions.report: stock.action_report_picking_packages
+DEL ir.actions.report: stock.stock_replenishment_report_product_product_action
+DEL ir.actions.report: stock.stock_replenishment_report_product_template_action
+NEW ir.actions.server: stock.action_revert_inventory_adjustment
+NEW ir.model.access: stock.access_product_tag_stock_manager
+NEW ir.model.access: stock.access_stock_lot_label_layout_user
+NEW ir.model.access: stock.access_stock_lot_user
+NEW ir.model.access: stock.access_stock_picking_label_type_user
+NEW ir.model.access: stock.access_stock_replenish_option
+DEL ir.model.access: stock.access_stock_production_lot_user
```

Comment:
renamed to stock.access_stock_lot_user

Reply:
Renamed via `rename_xmlids` in pre-migration script.